templates/base.html.twig

```diff
@@ -27,6 +27,13 @@
 			margin: -20px -20px 20px;
 		}
 
+		{% if is_rtl() %}
+		{# Hacky fix for RTL with Bootstrap 3. #}
+		.col-lg-1, .col-lg-2, .col-lg-3, .col-lg-4, .col-lg-5, .col-lg-6, .col-lg-7, .col-lg-8, .col-lg-9, .col-lg-10, .col-lg-11, .col-lg-12 {
+			float: right;
+		}
```

Comment:
you could also do:
```
{% if is_rtl() %}
<link rel="stylesheet" type="text/css" href="https://tools-static.wmflabs.org/cdnjs/ajax/libs/bootstrap-rtl/3.4.0/css/bootstrap-flipped.min.css">
{% endif %}
```
which should cover all of Bootstrap. But with as simple as WS Export is I think this 'hack' is probably sufficient.

Reply:
Thank you! I didn't know that existed.